gpt4all-backend/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ foreach(BUILD_VARIANT IN LISTS BUILD_VARIANTS)
 endforeach()
 
 add_library(llmodel
-    llmodel.h llmodel.cpp
+    llmodel.h llmodel.cpp llmodel_shared.cpp
```

Comment:
This was needed to allow chat.exe to compile as we subclass llmodel with chatgpt. otherwise, the compile was broken.

Still, need to test this on mac/windows...